clic/src/tier1/cleLaplaceBoxKernel.cpp

```diff
@@ -0,0 +1,28 @@
+
+#include "cleLaplaceBoxKernel.hpp"
+
+namespace cle
+{
+
+LaplaceBoxKernel::LaplaceBoxKernel(const ProcessorPointer & device)
+  : Operation(device, 2)
+{
+  std::string cl_header = {
+#include "cle_laplace_box.h"
+  };
+  this->SetSource("laplace_box", cl_header);
+}
```

Comment:
Library was updated, this need to be changed as followed:
- move the include outside the function, at the top of the file
- replace `cl_header` by `oclKernel::laplace_box`

see: [absolute kernel](https://github.com/clEsperanto/CLIc_prototype/blob/master/clic/src/tier1/cleAbsoluteKernel.cpp) for example